src/routes/user.ts

```diff
@@ -0,0 +1,65 @@
+import express from 'express'
+import { UserDocument } from '../utils/types/userType'
+import { UserService } from './../services/userService'
+import { Messages } from './../errors/mesages'
+
+const router = express.Router()
+
+router.post('/signup', async (req, res) => {
+
+    if (await UserService.tryToGetUser(req.body.email)) {
```

Comment:
A rather unclear name "tryToGetUser", maybe findUser would be better? What's your thoughts on this?